### PR TITLE
Discourse implementation

### PR DESCRIPTION
**Pull request by alexfoias**
Updated readme file with detailed information concerning the discourse forum.
Redirect forum page to discourse platform.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,64 +14,93 @@ https://www.taniarascia.com/make-a-static-website-with-jekyll/
...
+- Install Discourse
 ~~~
 ./discourse-setup
 Hostname      : forum.spinalcordmri.org
-Email         : xxx@spinalcordmri.org
-SMTP address  : smtp.elasticemail.com
-SMTP port     : 2525
-SMTP username : xxx  # get this from Elasticmail > Settings > SMTP/API
-SMTP password : pa$$word  # get this from Elasticmail > Settings > SMTP/API
+Email         : spinalcordmri@gmail.com
```
Comment (jcohenadad) [inline README.md]: i think we should remove this address (to minimize spamming)

File: README.md [hunk 2]
```diff
@@ -14,64 +14,93 @@ https://www.taniarascia.com/make-a-static-website-with-jekyll/
...
+### Setup subdomain in namecheap
+  - To create a subdomain, please do the following:
+    - Go to your Domain List and click Manage next to the domain
+    - Select the Advanced DNS tab
+    - Find the Host Records section and click on the Add New Record button
+    - Select A Record for Type and enter the Host `forum.spinalcordmri.org`  you would like to point to an IP address `DigitalOcean_Server_IP_Address`
+
+### Configuring Google login for Discourse ([reference](https://meta.discourse.org/t/configuring-google-login-for-discourse/15858))
+
+Using the spinalcordmri@gmail.com account, go to https://console.developers.google.com, click on Credentials and create a new Project.
```
Comment (jcohenadad) [inline README.md]: same comment about the email address

File: README.md (hunk 1 above)
Comment (alexfoias) [reply README.md]: Fixed in https://github.com/spinalcordmri/spinalcordmri.github.io/pull/15/commits/d555da77237e377e95765083bbac94596eeb3a76

File: README.md (hunk 2 above)
Comment (alexfoias) [reply README.md]: Fixed in https://github.com/spinalcordmri/spinalcordmri.github.io/pull/15/commits/d555da77237e377e95765083bbac94596eeb3a76

**Comment by alexfoias** [comment]
@jcohenadad I addressed your comments.

**Pull request merged**